fission/src/systems/scene/SceneRenderer.ts

```diff
@@ -236,13 +231,29 @@ class SceneRenderer extends WorldSystem {
         return id
     }
 
+    /** Registers gizmos that are attached to a parent mirabufsceneobject  */
+    public RegisterGizmoSceneObject(obj: GizmoSceneObject): number {
+        if (obj.HasParent()) this._gizmosOnMirabuf.set(obj.parentObjectId!, obj)
```

Comment:
Will `obj.parentObjectId` necessarily be defined?

Reply:
`obj.HasParent()` essentially checks `obj.parent != undefined`.

Reply:
I suppose you could technically add an unregistered object to the gizmo